promutils/scope.go

```diff
@@ -133,6 +133,11 @@ type Scope interface {
 	NewSummary(name, description string) (prometheus.Summary, error)
 	MustNewSummary(name, description string) prometheus.Summary
 
+	// Creates new prometheus.Summary metric with a custom set of objectives
```

Comment:
can you explain what an objective is in this comment? the link doesn't elaborate